### PR TITLE
Dashboard navigation

### LINKED ISSUE #83: Dashboard - add navigation bar
Add a navigation bar to the top of dashboard pages to make it easier to move around pages and more visually appealing.

As an engineer I need to ensure the dashboard is easy to navigate around by adding a navigation bar

### PR DESCRIPTION
**Pull request by alina-101**
Added a navigation bar instead of a side menu.
Also added headers and descriptions to introduce visualisations

closes #83

### CODE REVIEW

File: dashboard/pages/Home.py [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
 import streamlit as st
-
 import database
 import charts
 
 
-if __name__ == "__main__":
+def show_home():
 
     load_dotenv()
```
Comment (Zhi-704) [inline dashboard/pages/Home.py:12]: You may be calling load_dotenv multiple times, which is redundant, have it run once in your main function or when it runs

File: dashboard/pages/Tags.py [hunk 2]
```diff
@@ -2,15 +2,25 @@
 
 import streamlit as st
 
-from database import get_connection, get_tag_sales_by_tag, get_all_tag_names
-from charts import get_tag_sales_line_graph
-conn = get_connection()
```
Comment (Zhi-704) [inline dashboard/pages/Tags.py:7]: good, I agree that the conn shouldn't be a global variable

File: requirements.txt [hunk 3]
```diff
@@ -15,3 +15,4 @@ pytest-asyncio
 pytest-cov
 requests
 streamlit
+streamlit-navigation-bar
```
Comment (Zhi-704) [inline requirements.txt:18]: Good, updating the main requirements as well

**Comment by Zhi-704** [comment]
Quite happy with the code, doesn't seem to be anything sticking out.

File: dashboard/pages/Home.py (hunk 1 above)
Comment (alina-101) [reply dashboard/pages/Home.py:12]: Okay sounds good, I've moved it to `app.py`

File: dashboard/pages/Locations.py [hunk 4]
```diff
@@ -6,8 +6,10 @@
 from database import get_connection, get_sales_by_country
 from charts import create_choropleth_map
 
-if __name__ == "__main__":
```
[the remark this answers is not in the document]
Comment (alina-101) [reply dashboard/pages/Locations.py:9]: To introduce the navigation bar I changed the main function to a function I can call from another file

**Review by Zhi-704 — APPROVED** [review]
Happy with the changes!

**Pull request merged**